src/display.jl

```diff
@@ -15,9 +14,6 @@ end
 
 function Base.show(io::IO, mm::MIME"text/html", p::SyncPlot)
     # if we are rendering docs -- short circuit and display html
-    if get_renderer() == DOCS
```

Comment:
I think this function is used when generating the Documenter pages (not for Blink per se).